app/views/records/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<p id="notice"><%= notice %></p>
+
+<div class="card mb-3">
+  <div class="card-header">
+    <i class="fas fa-compact-disc"></i>
+    Records
+  </div>
+  <div class="card-body">
+    <div class="table-responsive">
+      <table class="table table-bordered" id="dataTable" width="100%" cellspacing="0">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @records.each do |record| %>
+            <tr>
+              <td><%= link_to record.name, record %></td>
+              <td class="text-center">
+                <%= link_to edit_record_path(record) do %>
+                  <i class="fas fa-pencil-alt"></i>
+                <% end %>
+                <%= link_to record, method: :delete, data: { confirm: 'Are you sure?' } do %>
+                  <i class="fas fa-times"></i>
+                <% end %>
+              </td>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+
+<br>
+
+<%= button_to "New Record", new_record_path, :method => "get", class: "btn btn-primary btn-lg btn-block login-button" %>
```

Comment:
```suggestion
<%= link_to(t('.new'), new_record_path, class: "btn btn-success float-right") %>
```

Reply:
I'm okay with using I18n more extensively but maybe we should focus on more basic things. We can, however, set the application correctly but don't enforce the use of i18n through the course. WDYT?

Reply:
Aham mas o resto acho que fica mais claro mesmo ser usar o `:method => "get"` não?!